src/main/java/com/bootcamp/demo/security/SecurityConfig.java

```diff
@@ -12,6 +12,7 @@
 
     @Override
     public void configure(HttpSecurity http) throws Exception {
+        /*
         http
```

Comment:
is this comment important or it can be removed ?

Reply:
is not important, it's useful when testing the endpoint so the login screen will not pop up